test/neura/fusion/test.mlir

```diff
@@ -0,0 +1,24 @@
+# RUN: clang++ -S -emit-llvm -O3 -fno-unroll-loops -fno-vectorize -o %t-kernel.ll kernel.cpp
```

Comment:
`-fno-unroll-loops -fno-vectorize` these two really matter? It would be vectorized or unrolled if you don't add them?

Reply:
It is only vectorized when I add `-f-vectorize` on my side. Anyways, LGTM. Thanks.